src/server/req_runjob.c

```diff
@@ -1084,7 +1073,11 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 			pjob->ji_qs.ji_jobid,
 			"Unable to Run Job, send to Mom failed");
 
-		if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_PROVISION)
+		if(pbs_errno == PBSE_NORELYMOM)
+			mark_node_down(pjob->ji_qs.ji_destin, "could not send job to mom");
```

Comment:
What happens when the server fails to connect to mom while staging in the files? It seems that svr_strtjob2 may not even get called in that case.

Reply:
I was actually following the failure cases in of `post_sendmom()` here. Now after looking at the code of `svr_connect()` I think You are right that I might be marking the node down again.  I will remove this `mark_node_down()` call in the next push.